### ISSUE TITLE
How to repeat a tensor N times to have same shape with a dynamic tensor?

### ISSUE THREAD

**Issue by charliedream1** [NVIDIA/TensorRT#2409]
## Description

I have a tensor A with shape of (1,3,512), and a tensor B with dynamic shape (-1,3,512), -1 is the dynamic shape
how to repeat copy tensor A on axis 0 to have same shape with tensor B, so that I can make adding tensor A and B?

With python API, how to get dimension with "add_shape" API to get dim of axis 0, and use "loop" to make "concat",
could you give an example? The process is so much complicated

## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by zerollzeng** [comment]
it's duplicated with #2408 , so I'm closing this.
